### ISSUE TITLE
git worktree support?

### ISSUE THREAD

**Issue by ashnur** [brujoand/sbp#125]
**Describe the bug**
Git segment not visible in worktree folders

**To Reproduce**
1. Go to a folder created by git worktree
2. See that git segment is missing

**Expected behavior**
Git segment should be visible in git worktree folders

**Screenshots**
![image](https://user-images.githubusercontent.com/407934/158325274-7ee5aa83-6a0d-4f69-8701-99ac50bb140f.png)

**Environment (please complete the following information):**
 - OS: WSL2
 - Terminal: https://github.com/Microsoft/Terminal
 - Version: v0.3

**Comment by brujoand** [comment]
Yeah I haven't used git worktree myself, so support for this would have to be added. Feel free to give it a try!
